dired-filter.el

```diff
@@ -393,6 +393,20 @@ See `dired-filter-stack' for the format of FILTER-STACK."
   "The face used to highlight pair overlays."
   :group 'dired-filter-group)
 
+(defvar dired-filter-group-imenu-generic-expression '("Group" "^ *\\[\\(.*\\)\\]" 1)
+  "An alist of menus for accessing locations in documents with imenu
+
+It is a list of lists of the form
+(MENU-TITLE REGEXP INDEX [FUNCTION] [ARGUMENTS...]
+
+See `imenu-generic-expression' for more information.
+)
```

Comment:
I think this paren belongs somewhere else, after the `[ARGUMENTS...]`?

Reply:
That's right